controllers/users_controller.php

```diff
@@ -0,0 +1,39 @@
+<?php
+ob_start();
+get_model('user');
+
+/*
+ * This class inherits from the base class and contains the calls to the posts procedures
+ */
+class UsersController extends User  {
+    private $params;
+    
+    public function __construct($params) {
+        var_dump($params);
```

Comment:
if you use this to debug, now that will be part of the main branch it would be unnecessary beacause you already deal with all the errors, or so i hope...